src/Common/src/CoreLib/System/IO/PathInternal.Windows.cs

```diff
@@ -401,15 +401,7 @@ internal static string NormalizeDirectorySeparators(string path)
         /// </summary>
         internal static bool IsEffectivelyEmpty(ReadOnlySpan<char> path)
         {
-            if (path.IsEmpty)
-                return true;
-
-            foreach (char c in path)
-            {
-                if (c != ' ')
-                    return false;
-            }
-            return true;
+            return (path.IsEmpty || path.IndexOf(' ') > -1);
```

Comment:
This isn't right.  Previously this would return `false` if any character in `path` wasn't ' '.  Now it'll return true in such a situation as long as there's at least one ' ' in `path`.